mapzen-places-api/src/main/java/com/mapzen/places/api/ui/PlaceAutocomplete.java

```diff
@@ -0,0 +1,105 @@
+package com.mapzen.places.api.ui;
+
+import com.mapzen.android.lost.api.Status;
+import com.mapzen.places.api.AutocompleteFilter;
+import com.mapzen.places.api.LatLngBounds;
+import com.mapzen.places.api.Place;
+import com.mapzen.places.api.internal.PlaceAutocompleteActivity;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+
+import static com.mapzen.places.api.internal.PlaceIntentConsts.EXTRA_BOUNDS;
+import static com.mapzen.places.api.internal.PlaceIntentConsts.EXTRA_PLACE;
+
+/**
+ * Main entry point for the autocomplete API. A search widget is presented and users can select
+ * autocomplete results. Unlike the {@link PlacePicker} API, a map is not shown.
+ */
+public class PlaceAutocomplete {
+
+  public static final int MODE_FULLSCREEN = 1;
+  public static final int MODE_OVERLAY = 2;
+
+  public static final String EXTRA_STATUS = "extra_status";
+  public static final String EXTRA_FILTER = "extra_filter";
```

Comment:
Would these be better in `PlaceIntentConsts` to avoid adding them to the public API?